src/StateService/Settings.cs

```diff
@@ -6,14 +6,16 @@ internal class Settings
     {
         public string Path { get; }
         public bool FullState { get; }
+        public uint Network { get; }
         public bool AutoVerify { get; }
 
         public static Settings Default { get; private set; }
 
         private Settings(IConfigurationSection section)
         {
-            Path = string.Format(section.GetValue("Path", "Data_MPT_{0}"), ProtocolSettings.Default.Magic.ToString("X8"));
+            Path = section.GetValue("Path", "Data_MPT_{0}");
```

Comment:
require format or remove {0}

Reply:
It is formatted here.
https://github.com/neo-project/neo-modules/blob/10e110ea4c6ec46c77d2f5b03fa07fb968903198/src/StateService/StatePlugin.cs#L44